react-app/src/library/services/vendor-implementations/plantronics/plantronics.ts

```diff
@@ -62,20 +87,28 @@ export default class PlantronicsService extends VendorImplementation {
   }
 
   async pollForCallEvents() {
+    if (this.callEventsTimerId) {
+      clearTimeout(this.callEventsTimerId);
+      this.callEventsTimerId = null;
+    }
     if (this.isConnected && this.isActive && !this.disableEventPolling) {
       await this.getCallEvents();
     }
-    setTimeout(() => {
+    this.callEventsTimerId = setTimeout(() => {
```

Comment:
The use of timeouts has me a little nervous because of Chrome's intensive throttling... I don't have a lot of background on the polling system used here, though.